ChemistryLib/PhreeqcInterfaceData/Solution.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <boost/optional/optional.hpp>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "BaseLib/Error.h"
+
+namespace BaseLib
+{
+class ConfigTree;
+}
+
+namespace ChemistryLib
+{
+struct Component
+{
+    Component(std::string name) : _name(name) {}
```

Comment:
when passing by value, don't forget to move the argument:
```c++
Component(std::string name) : _name(std::move(name)) {}
```

Reply:
Fixed. Also in others (phreeqcIO, Output, OutputItem, Solution, Component, ReactionRate, KineticReactant, and EquilibriumPhase).